### PR TITLE
@TEST-155 @TEST-188 test case fix along with review comments

### PR DESCRIPTION
**Pull request by jaind1046**
@TEST-155 test case fix along with review comments

@TEST-Change-all-content-flag-to-sanitise-for-all-doc-type --> it not given sanitise in the scenario file as it is already covered in @TEST-155

### CODE REVIEW

**Comment by jaind1046** [comment]
As the video was big in size to upload, So uploaded the screenshot of test cases run on local.

**Comment by jaind1046** [comment]
TEST-155

File: specs/features/content-management-policy-editing.feature [hunk 1]
```diff
@@ -21,28 +21,35 @@ Feature: Content Management Policy Editing
...
+        Examples:`
+            | fileType   | ContentFlags         | FlagType |
+            | word       | embeddedFiles        | sanitise |
+            | excel      | externalHyperlinks   | disallow |
+            | powerpoint | metadata             | disallow |
+            | pdf        | actionsAll           | sanitise |
 
 
-    @TEST-Change-all-content-flag-for-all-doc-type
+    @TEST-Change-all-content-flag-to-sanitise-for-all-doc-type
```
Comment (NadHodeGW) [inline specs/features/content-management-policy-editing.feature:40]: Please make these 2 scenarios: "all-content-flag-to-sanitise" & "all-content-flag-to-disallow" into one and add the result column for saved flags

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
comment in file

**Comment by jaind1046** [comment]
change all content flag

File: specs/features/content-management-policy-editing.feature (hunk 1 above)
Comment (jaind1046) [reply specs/features/content-management-policy-editing.feature]: Done. Thank you.

**Comment by jaind1046** [comment]
TEST-188-updated

**Comment by jaind1046** [comment]
Tested TC-188 after cahnge

**Review by NadHodeGW — APPROVED** [review]

**Pull request merged**